commcare/src/main/java/org/motechproject/commcare/service/impl/CommcareConfigServiceImpl.java

```diff
@@ -164,6 +164,16 @@ public boolean verifyConfig(Config config) {
     }
 
     @Override
+    public Config syncConfig(Config config) {
+
+        eventRelay.sendEventMessage(new MotechEvent(EventSubjects.CONFIG_UPDATED, prepareParams(config.getName())));
+        configs.updateConfig(config);
+        updateConfigs();
```

Comment:
Why do we need the config updates here? Wouldn't sending an event be sufficient?